### PR TITLE
Make validation account for /prisoner in preprod

### PR DESCRIPTION
**Pull request by tatyree**
https://preprod-prisonvisits.dsd.io 301s to production.  You can only
view the preprod site if you hit it on
https://preprod-prisonvisits.dsd.io/prisoner.  This breaks the smoke
test. I thought it would be enough to check that the URL had the correct
path in it somewhere.

### CODE REVIEW

**Comment by SteveMarshall** [comment]
To clarify: this is a safeguard against someone inadvertently starting smoke-testing against `/` instead of `/prisoner`, right?

**Comment by tatyree** [comment]
Not exactly.  The smoke test was hitting /prisoner.  This caused the first test to fail on preprod, which responds on /prisoner/prisoner.

File: smoke_test/steps/prisoner_page.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module SmokeTest
   module Steps
     class PrisonerPage < BaseStep
-      PAGE_PATH = '/prisoner'
+      PAGE_PATH = %r{/prisoner$}
```
Comment (dan-palmer) [inline smoke_test/steps/prisoner_page.rb]: I don't think this is correct - should expect the url to be something like: `https://www.prisonvisits.service.gov.uk/prisoner`

**Comment by tatyree** [comment]
Not needed.  `/prisoner` was being accidentally included in the host environment variable.

**Pull request closed without merging**